ops/pebble.py

```diff
@@ -1028,6 +1056,48 @@ class CheckStatus(enum.Enum):
     DOWN = 'down'
 
 
+class LogTarget:
+    """Represents a log target in a Pebble configuration layer."""
+
+    def __init__(self, name: str, raw: Optional['LogTargetDict'] = None):
+        self.name = name
+        dct: LogTargetDict = raw or {}
+        self.override: str = dct.get('override', '')
+        self.type_ = dct.get('type', '')
+        self.location = dct.get('location', '')
+        services = dct.get('services')
+        if services:
+            services = services[:]
```

Comment:
Out of interest, why this vs the pattern we use elsewhere, like for `Service.after` for example:

```python
        self.after = list(dct.get('after', []))
```

I also tend to think `list(x)` is more explicit than `x[:]` -- don't have strong feelings, but we should probably be consistent in the same file.

Reply:
> Out of interest, why this vs the pattern we use elsewhere, like for `Service.after` for example:
> 
> ```python
>         self.after = list(dct.get('after', []))
> ```
> 
> I also tend to think `list(x)` is more explicit than `x[:]` -- don't have strong feelings, but we should probably be consistent in the same file.

Hah, in this case it's because I was not copying what was done elsewhere and was just writing code :joy:. I agree `list(x)` is more explicit than `x[:]` - I'm not sure why I default to using `[:]` - maybe because it used to be quite a bit faster? Maybe it was just a habit I picked up somewhere. I would say that `x.copy()` is probably the *most* explicit (and it also seems to be the fastest now), but I agree that consistency in the file trumps everything else, so will adjust to that, thanks!